src/doc/src/guide/creating-a-new-workspace.md

```diff
@@ -0,0 +1,75 @@
+# Creating a New Workspace
+
+A [workspace][def-workspace] is a collection of one or more packages, called workspace members, that are managed together.
```

Comment:
In Cargo's documentation, we usually either wrap at column like 80/120, or at where makes sense and natural like punctuations or conjunctions.

Reply:
https://rhodesmill.org/brandon/2012/one-sentence-per-line/